packages/webapp/src/components/Navigation/Menus/ProfileMenu.jsx

```diff
@@ -0,0 +1,161 @@
+    isMobile ? setIsBottomDrawerOpen(false) : closeFloater();
+  };
+
+  const logOutClick = () => {
+    logout();
+    closeMenu();
+  };
+
+  const openTutorialsClick = () => {
+    const playlistIDs = {
+      es: 'PLDRpVZ4VsXJhghxfEQuApFQTeCWUbGBN9',
+      pt: 'PLDRpVZ4VsXJg0ke20m47MmJq6uAJAlAGF',
+      en: 'PLDRpVZ4VsXJgVGrmmXJooNqceXvre8IDY',
+    };
+
+    const playList = playlistIDs[selectedLanguage] || playlistIDs['en'];
+    const url = 'https://www.youtube.com/playlist?list=' + playList;
+
+    const win = window.open(url, '_blank');
+    win.focus();
+    closeMenu();
+  };
+
+  const handleClick = (link) => {
+    closeMenu();
+    history.push(link);
+  };
+  const options = [
+    {
+      id: 'user-profile',
+      onClick: () => handleClick('/profile'),
+      icon: <MyInfoIcon />,
+      label: t('PROFILE_FLOATER.INFO'),
+      externalLink: false,
+    },
+    {
+      id: 'farm-selection',
+      onClick: () => handleClick('/farm_selection'),
+      icon: <SwitchFarmIcon />,
+      label: t('PROFILE_FLOATER.SWITCH'),
+      externalLink: false,
+    },
+    {
+      id: 'help',
+      onClick: () => handleClick('/help'),
+      icon: <HelpIcon />,
+      label: t('PROFILE_FLOATER.HELP'),
+      externalLink: false,
+    },
+    {
+      id: 'tutorials',
+      onClick: openTutorialsClick,
+      icon: <VideoIcon />,
+      label: t('PROFILE_FLOATER.TUTORIALS'),
+      externalLink: true,
+    },
+    {
+      id: 'logout',
+      onClick: logOutClick,
+      icon: <LogoutIcon style={{ transform: 'translateX(2px)' }} />,
+      label: t('PROFILE_FLOATER.LOG_OUT'),
+      externalLink: false,
+    },
+  ];
+  const menuItems = options.map((option) => {
+    const { id, onClick, icon, label, externalLink } = option;
+    return (
+      <MenuItem key={id} onClick={onClick} classes={{ root: styles.menuItemRoot }}>
+        <ListItemIcon>{icon}</ListItemIcon>
+        <ListItemText classes={{ root: styles.itemTextRoot }}>{label}</ListItemText>
+        {externalLink && <LaunchIcon />}
+      </MenuItem>
+    );
+  });
+
+  const plainMenu = (
+    <Menu
+      id="profile-menu"
+      anchorEl={target.current}
+      open={open}
+      onClose={onClose}
+      MenuListProps={{
+        'aria-labelledby': 'profile-navigation-button',
+      }}
+      classes={{ list: styles.menuList, root: styles.menuRoot }}
+    >
+      {menuItems}
+    </Menu>
+  );
+
+  const drawerMenu = (
+    <MenuList
+      id="profile-menu"
+      open={open}
+      onClose={onClose}
+      MenuListProps={{
+        'aria-labelledby': 'profile-navigation-button',
+      }}
+      disablePadding
+      classes={{ list: styles.drawerMenuList, paper: styles.drawerMenuPaper }}
+      alignItems="flex-end"
+    >
+      <ListSubheader classes={{ root: styles.drawerListSubheader }}>
+        <CloseX onClick={() => setIsBottomDrawerOpen(false)} />
+      </ListSubheader>
+      {menuItems}
+    </MenuList>
+  );
+
+  return isMobile ? (
+    <SwipeableDrawer
```

Comment:
I'd suggest using our own `Drawer` component instead of the one from MUI since it already includes  the X to close and some styling to make the height and scrolling right

Reply:
@antsgar Yeah I took a look into doing that but then I second guessed myself because it is a custom component that I would need to adapt-- and I can do that no big deal. So question from me is: Is the direction we should be heading to use more components from our MUI library or adapt our custom ones? 

Adaptions I am making to our custom are small: close x button -- is different so I would need to create an option for that. And possibly also the padding for children: 
![Screenshot 2023-12-19 at 5 11 09 PM](https://github.com/LiteFarmOrg/LiteFarm/assets/30075973/20f60033-c5b8-4d37-ae89-b1537ff40a8c)
![Screenshot 2023-12-19 at 5 55 50 PM](https://github.com/LiteFarmOrg/LiteFarm/assets/30075973/50ab0175-6b07-4fe4-88ad-5d0753edb257)


Potential future adaptions: If sometime in the future our UI/UX wants both a left anchored drawer and a bottom drawer, or to have theming capabilities for dark/light mode. It could get increasingly complex.

I think my personal preference it to let MUI do the hard work since they are the experts and have a lot of functionality and delete our custom components wherever possible. But its no problem for either way.

If we like the MUI components I still need to change this line: it should not be a SwipeableDrawer thats more for main menu stuff I think.

Reply:
Ideally I think we'd do a little bit of both -- have our own custom component built on top MUI's one. That way, we'd let MUI do the hard work if we don't want to reinvent the wheel, but we'd have an extra layer of abstraction on top that would let us either expand or limit that functionality so that it fits the UX patterns of our app. However I don't think we need to address this right now, I'm cool with using MUI's drawer for this one and eventually refactoring our Drawer component.

Reply:
Yeah that makes sense to me as the ideal situation. I guess I didn't want to/didn't know if I should refactor Drawer haha Thanks!